AmplifyPlugins/API/AWSAPICategoryPlugin/AWSAPIPlugin+Resettable.swift

```diff
@@ -25,6 +25,10 @@ extension AWSAPIPlugin: Resettable {
 
         authService = nil
 
+        if #available(iOS 13.0, *) {
```

Comment:
See below, I think we can simplify this and do away with the availability check

Reply:
Sorry, I was unable to figure out what you were suggesting.